.byebug_history

```diff
@@ -1,2 +1,8 @@
 exit
+item
+item.slug
+exit
+business.items.first.slug
+business.items.first
+exit
 params
```

Comment:
not sure why this stuff is here...we should probably add to gitignore file. 

since it is already up here , run:
```
git rm -r .byebug_history
```
that'll remove it from your commits.